bin/kuzzle

```diff
@@ -139,6 +138,28 @@ bindOption(
     .action(require('./commands/loadSecurities'))
 );
 
+// $ kuzzle encryptSecrets
+bindOption(
+  program
+    .command('encryptSecrets <file>')
+    .option('    --vault-key <vaultKey>', 'Vault key used to encrypt secrets')
```

Comment:
I just spotted that during my review of this feature's documentation: you're mixing snake-case with camelCase.
These options should be named `vaultKey`, `secretsFile`, and so on.

Reply:
I do this because `vaultKey` will be the variable name used in the script but usually command line options are in snake-case

Reply:
I know, and commands should be in snake-case too for the sake of convention... but it's too late now: we have `--noColors` and commands are all camelCased. :-(

Reply:
I won't block that PR because of this, though

Reply:
Oh.. I think we can get ride of it in v2